lsp-mode.el

```diff
@@ -5019,11 +5019,13 @@ It will show up only if current point has signature help."
             (active-signature? (or lsp--signature-last-index active-signature? 0))
             (_ (setq lsp--signature-last-index active-signature?))
             ((signature &as &SignatureInformation? :label :parameters?) (seq-elt signatures active-signature?))
-            (prefix (concat (propertize (format " %s/%s"
-                                                (1+ active-signature?)
-                                                (length signatures))
-                                        'face 'success)
-                            " │ "))
+            (prefix (if (eq 1 (length signatures))
```

Comment:
`(not (cl-rest signatures))` is the idiomatic way to check if the size is 1.

Reply:
Is that faster? It does not look easier to understand to me :/

Reply:
`length` will iterate over the list.

Reply:
Yes, it's much faster, `O(1)` instead of `O(n)` like using `length`.
``` elisp
(length tmp)
91803

(benchmark-run 1000 (length tmp))
(0.665376 0 0.0)

(benchmark-run 1000 (cl-rest tmp))
(0.0002550000000000001 0 0.0)
```

Reply:
Good to know, thanks